src/database.rs

```diff
@@ -0,0 +1,37 @@
+use diesel::prelude::*;
+use diesel::pg::PgConnection;
+use dotenv::dotenv;
+use std::env;
+use message::Message;
+use diesel::insert;
+
+pub fn connect() -> PgConnection {
+    dotenv().ok();
+
+    let database_url = env::var("DATABASE_URL").expect("DATABASE_URL must be set");
+    PgConnection::establish(&database_url).expect(&format!("Error connecting to {}", database_url))
+}
+
+
+pub fn insert_message(connection: &PgConnection,
+                      id: u64,
+                      content: String,
+                      guild_id: u64,
+                      author_id: u64,
+                      channel_id: u64)
+                      -> Message {
+    use schema::message;
+
+    let new_message = Message {
+        id: id as i64,
+        content: content,
+        guild_id: guild_id as i64,
+        author_id: author_id as i64,
+        channel_id: channel_id as i64,
+    };
+
+    insert(&new_message)
+        .into(message::table)
+        .get_result(connection)
+        .expect("Could not insert message")
```

Comment:
You sure you want to `.expect()` and risk a panic in the event of DB error?

Normally this is the sort of thing you just log.